.github/workflows/issues-workflow.yaml

```diff
@@ -11,6 +11,7 @@ on:
 jobs:
   add-to-project:
     name: Add issue to project
+    if: github.event.pull_request.head.repo.full_name == github.repository  # Only run jobs if the feature branch is in your repo (not in a fork)
```

Comment:
But what happens on `issue.opened` then? Do we need to add an `or` clause there?

Reply:
good catch. I will investigate a bit further.

Reply:
the other clause linked with `OR` should be checking the author association regarding the issue's project. There is indeed some context property about that: `github.event.issue.author_association` documented [here](https://docs.github.com/en/rest/using-the-rest-api/issue-event-types?apiVersion=2022-11-28).

Before going ahead, I would like to hear from you if that is the correct approach or we should rely on the existence of the env var and that's it?

Reply:
Got it, we want issues from any user to be added to the project